firefox_ui_tests/remote/security/test_dv_certificate.py

```diff
@@ -0,0 +1,103 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette_driver import By, Wait
+
+from firefox_ui_harness.decorators import skip_under_xvfb
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+class TestDVCertificate(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+        self.test_url = "https://ssl-dv.mozqa.com"
+        self.cert = None
+        self.locationbar = self.browser.navbar.locationbar
+        self.identity_popup = self.browser.navbar.locationbar.identity_popup
+
+    def tearDown(self):
+        self.marionette.set_context("chrome")
+        self.identity_popup.close(force=True)
+        FirefoxTestCase.tearDown(self)
+
+    @skip_under_xvfb
+    def test_larry_blue(self):
```

Comment:
Lets rip off `larry` from all our tests. It's misleading and hard to understand if you don't have the history. So just take DV here.